src/scss/CreateRun.scss

```diff
@@ -14,6 +14,14 @@ limitations under the License.
 @import '~carbon-components/scss/globals/scss/vars';
 
 .create-pipelinerun {
+  .bx--form__helper-text {
+    color: #6f6f6f; // Since by default the contrast/colour choice is barely readable.
```

Comment:
`color: $text-05;`

Reply:
let's also override the max-width here so it doesn't create such a blocky looking piece of text, 100% is fine